src/main/java/nextstep/ladder/controller/RandomLineSelector.java

```diff
@@ -0,0 +1,17 @@
+package nextstep.ladder.controller;
+
+import nextstep.ladder.domain.LineSelector;
+
+import java.util.Random;
+
+public class RandomLineSelector implements LineSelector {
+    private Random random;
+
+    public RandomLineSelector() {
+        random = new Random();
+    }
+
+    @Override public int select(int size) {
```

Comment:
```suggestion
     @Override
     public int select(int size) {
```
어노테이션은 메소드 선언부 위로 올려주세요!

Reply:
다른거 찾다가 방금 발견한게 있습니다.[ google style guide](https://google.github.io/styleguide/javaguide.html#s4.8.5-annotations) 에 의하면 파라메터가 없는 하나의 어노테이션은 함수 선언과 같은 줄에 선언할 수 있도록 예외케이스를 두었습니다. 함께 알면 좋을것 같아 공유해봅니다 😃